config/config.go

```diff
@@ -121,6 +123,23 @@ func GitHubRemoteSource(config *Config, gitcmd git.GitInterface) *remoteSource {
 	}
 }
 
+func MergeMethod(config *Config) (mergeMethod githubv4.PullRequestMergeMethod, err error) {
```

Comment:
make this a member function of config:
`func (c *Config)MergeMethod (githubv4.PullRequestMergeMethod, error) {`

also, personal preference, I'm not a big fan of named return arguments, I prefer a simple:
  ` return githubv4.PullRequestMergeMethodMerge, nil`